### PR TITLE
Summarize ref matches

### PR DESCRIPTION
**Pull request by deflaux**
Update the schema for the multisample variants table to better accommodate larger cohorts.  Fixes https://github.com/googlegenomics/codelabs/issues/52

1. updated the dataflow job to emit the data for the new schema
1. deleted the hadoop streaming job instead of updating it
1. updated some SQL unnecessarily querying the multisample variants table when the genome call table was sufficient
1. updated other SQL specifically making use of the reference calls in the multisample variants table to work with the new schema

### CODE REVIEW

File: Java/PlatinumGenomes-variant-transformation/README.md [hunk 1]
```diff
@@ -84,19 +88,21 @@ java -cp target/non-variant-segment-transformer-*runnable.jar \
 
 ## Results
 
-You have now created a table like [google.com:biggene:platinum_genomes.transformed_variants](https://bigquery.cloud.google.com/table/google.com:biggene:platinum_genomes.transformed_variants?pli=1)
+You have now created a table like [google.com:biggene:platinum_genomes.transformed_variants](https://bigquery.cloud.google.com/table/google.com:biggene:platinum_genomes.multisample_variants?pli=1)
+
+Note that this schema is optimized for a large cohort which would have many, many more rare variants. We summarize the of callsets that match the reference for each variant instead of adding those individual genotypes for those callsets to the record.
```
Comment (mbookman) [inline Java/PlatinumGenomes-variant-transformation/README.md]: "We summarize the of callsets"?

File: Java/PlatinumGenomes-variant-transformation/src/main/java/com/google/cloud/genomics/examples/TransformNonVariantSegmentData.java [hunk 2]
```diff
@@ -164,37 +179,53 @@ private static TableSchema getTableSchema() {
...
-            (0 < alleleNumber) ? (genotypeCount.count(j) / (double) alleleNumber) : 0.0);
+      for (int j = 0; j < numAlts; j++) {
+        alts.add(new TableRow()
+            .set("alternate_bases", v.getAlternateBases(j))
+            .set(ALLELE_COUNT_FIELD, genotypeCount.count(j + 1))
+            .set(ALLELE_FREQUENCY_FIELD,
+                (0 < alleleNumber) ? (genotypeCount.count(j + 1) / (double) alleleNumber) : 0.0));
       }
       
+
```
Comment (mbookman) [inline Java/PlatinumGenomes-variant-transformation/src/main/java/com/google/cloud/genomics/examples/TransformNonVariantSegmentData.java]: Is this extra blank line intentional?

File: Java/PlatinumGenomes-variant-transformation/src/main/java/com/google/cloud/genomics/examples/TransformNonVariantSegmentData.java [hunk 3]
```diff
@@ -164,37 +179,53 @@ private static TableSchema getTableSchema() {
...
+      // Compute AN/AC/AF for SNPs.  Note that no-calls (genotype -1) are excluded from AN.
+      List<TableRow> alts = new ArrayList<>();
       int numAlts = v.getAlternateBasesCount();
       int alleleNumber = 0;
-      List<Integer> alleleCount = new ArrayList<Integer>(numAlts);
-      List<Double> alleleFrequency = new ArrayList<Double>(numAlts);
-      for (int i = 0; i <= numAlts; i++) {
-        alleleNumber += genotypeCount.count(i);
+      if (VariantUtils.IS_SNP.apply(v)) {
+        for (int i = 0; i <= numAlts; i++) {
```
Comment (mbookman) [inline Java/PlatinumGenomes-variant-transformation/src/main/java/com/google/cloud/genomics/examples/TransformNonVariantSegmentData.java]: Please verify that <= is correct here (versus "<").

**Comment by mbookman** [comment]
Other than a few small things, LGTM.

File: Java/PlatinumGenomes-variant-transformation/src/main/java/com/google/cloud/genomics/examples/TransformNonVariantSegmentData.java (hunk 3 above)
Comment (deflaux) [inline Java/PlatinumGenomes-variant-transformation/src/main/java/com/google/cloud/genomics/examples/TransformNonVariantSegmentData.java]: added a comment to explain why <= is needed

File: Java/PlatinumGenomes-variant-transformation/src/main/java/com/google/cloud/genomics/examples/TransformNonVariantSegmentData.java (hunk 2 above)
Comment (deflaux) [inline Java/PlatinumGenomes-variant-transformation/src/main/java/com/google/cloud/genomics/examples/TransformNonVariantSegmentData.java]: removed!

File: Java/PlatinumGenomes-variant-transformation/README.md (hunk 1 above)
Comment (deflaux) [inline Java/PlatinumGenomes-variant-transformation/README.md]: re-wrote that sentence, added a bit more detail

File: R/PlatinumGenomes-QC/Sample-Level-QC.md [hunk 4]
```diff
@@ -565,67 +565,106 @@ result <- DisplayAndDispatchQuery("./sql/homozygous-variants.sql",
 
 ```
 # Compute the expected and observed homozygosity rate for each individual.
+# FROM DOCUMENTATION IN VCFTOOLS
```
Comment (pgrosu) [inline R/PlatinumGenomes-QC/Sample-Level-QC.md]: Maybe adding the following link would help folks go to the specific line in the code that this references:

https://sourceforge.net/p/varianttools/git/ci/master/tree/variant_tools/utils.cpp#l326

**Pull request merged**